src/components/AboutUs/ClientTestimonial.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Padding } from 'styled-components-spacing'
+
+import { Row, Col, Grid } from '../grid'
+import { SectionTitle } from '../Typography'
+import GreyBackground from '../Common/GreyBackground'
+
+const Video = styled.iframe`
```

Comment:
Could we amalgamate this & the approach for the TemplatedCaseStudy, where we use a specially created `VideoSection` component:

https://github.com/yldio/yld.io/blob/875de550c7704cbbde32d21e37e0f9e78a7d1475/src/components/TemplatedCaseStudy/VideoSection.js#L5-L19

Reply:
I don't think this is justifiable...

the layout (num of cols) is different, in that section they map over an array whereas here I don't. The only thing that's similiar is really the `iframe` code. 

Also, I followed the same pattern that's used on the `talks.js` file — which I like more.

Reply:
Hmm I see what you mean. Tbh I have no idea why this is mapping over anything. We only want to show one video on the case study pages:
![image](https://user-images.githubusercontent.com/15656538/53163605-5eb18600-35c6-11e9-810f-853b190ab7f6.png)

and the reason I'm mentioning it is because it looks identical to what we have on this page.

So maybe the point is less "amalgamate approaches" & more of a request to remove `VideoSection`, use what you've done here instead and put it into Common. What do you think?

Reply:
I don't that much time to do it rn, but if you want to do it, please do :-)

Reply:
Kk :+1: